WBooks/Books Table/BookTableViewCell.swift

```diff
@@ -16,24 +16,12 @@ class BookTableViewCell: UITableViewCell, NibLoadable {
     @IBOutlet weak var logoImageView: UIImageView!
     @IBOutlet weak var nameLabel: UILabel!
     @IBOutlet weak var authorLabel: UILabel!
-
-    // MARK: - Initializers
-    
-    override func awakeFromNib() {
-        super.awakeFromNib()
-    }
-
-    override func setSelected(_ selected: Bool, animated: Bool) {
-        super.setSelected(selected, animated: animated)
-
-        // Configure the view for the selected state
-    }
     
     // MARK: - Helper methods
     
-    func configureCell() {
-        nameLabel.text = "This is the title and is so large"
-        authorLabel.text = "And this is the author"
-        logoImageView.image = UIImage(named: "image")
+    func configureCell(with book: Book) {
+        nameLabel.text = book.title
+        authorLabel.text = book.author
+        logoImageView.image = UIImage(named: book.imageUrl) // TODO: Download image when getting the URL from API
```

Comment:
It is better to be prepared for downloading from a url, even though at this point it won't come from api, but it will be a fixed URL (for example, one from google images you like)

For this, you can check the `ImageFetcher` in [Wolmo Utils](https://github.com/Wolox/wolmo-utils-ios) ;)